### PR TITLE
[Artwork] When in auction, show ‘bid now’ CTA instead.

### PR DESCRIPTION
**Pull request by alloy**
Part of #282.

~~Might still need a paddle icon, but will do so in another PR.~~

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/66d4e860d39bd4d1852dc0c3f8e0341a48f642f0/lib/components/artwork_grids/artwork.js">lib/components/artwork_grids/artwork.js</a>, <a href="https://github.com/artsy/emission/blob/66d4e860d39bd4d1852dc0c3f8e0341a48f642f0/lib/components/artwork_grids/generic_grid.js">lib/components/artwork_grids/generic_grid.js</a> &amp; <a href="https://github.com/artsy/emission/blob/66d4e860d39bd4d1852dc0c3f8e0341a48f642f0/lib/components/artwork_grids/infinite_scroll_grid.js">lib/components/artwork_grids/infinite_scroll_grid.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by alloy** [comment]
simulator screen shot 28 sep 2016 21 34 01

/cc @katarinabatina

**Comment by sarahscott** [comment]
There are some conflicts - should I fix them and merge or @alloy is there anything you want to do instead?

**Comment by alloy** [comment]
I'll rebase 👍

**Comment by alloy** [comment]
@sarahscott Done!

**Comment by sarahscott** [comment]
Merging!

File: lib/components/artist/artworks/index.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ class Artworks extends React.Component {
         <SerifText style={styles.heading}>
           <SerifText style={styles.text}>{title}</SerifText> <SerifText style={[styles.text, styles.count]}>({count})</SerifText>
         </SerifText>
-        <ArtworksGrid artworks={artworks} queryForState={stateQuery} queryForPage={pageQuery} onComplete={onComplete} queryArtworksKeypath="artist" />
+        <ArtworksGrid artworks={artworks} queryForState={stateQuery} queryForPage={pageQuery} onComplete={onComplete} queryArtworksKeypath="artist.artworks" />
```
Comment (alloy) [inline lib/components/artist/artworks/index.js]: cc @orta

**Pull request merged**